etc/kayobe/inventory/group_vars/network/defaults

```diff
@@ -0,0 +1,16 @@
+---
+# create network group
+
+ansible_user: "{{ kayobe_ansible_user }}"
+bootstrap_user: "{{ controller_bootstrap_user }}"
+lvm_groups: "{{ controller_lvm_groups }}"
+sysctl_parameters: "{{ controller_sysctl_parameters }}"
+users: "{{ controller_users }}"
+
+network_interfaces: >
```

Comment:
This list seems to be ignored for some reason :( I am getting the list for the controllers, which is trying to give out a public IP.

Reply:
Presumably it works if you move this out of defaults to the specific groups?

Reply:
seems to be picked up:
```
}[will@cumulus-seed kayobe-config]$ kayobe configuration dump --var-name network_interfaces --limit network
Using Kayobe config from /home/will/kayobe-env/src/kayobe-config
{
    "net1": [
        "admin_oc_net", 
        "bond0_net", 
        "br_net", 
        "external_net", 
        "tunnel_net", 
        "internal_net"
    ], 
    "net2": [
        "admin_oc_net", 
        "bond0_net", 
        "br_net", 
        "external_net", 
        "tunnel_net", 
        "internal_net"
    ]
```

Reply:
I don't see what is wrong here, running: `kayobe playbook run ~/kayobe-env/src/kayobe/ansible/ip-allocation.yml --limit network` leaves `network-allocation.yml` unchanged i.e no public IP :-/